applications/luci-app-nft-qos/luasrc/model/cbi/nft-qos/nft-qos.lua

```diff
@@ -226,4 +224,54 @@ o.default = '?'
 
 end
 
+--
+-- limit speed by mac address
+--
+o = s:taboption("limitmac", Flag, "limit_mac_enable", translate("Limit Enable"), translate("Enable Limit Rate Feature"))
+o.default = limit_mac_enable or o.enabled
+o.rmempty = false
+
+--
+-- Static By Mac Address
+--
+if limit_mac_enable == "1" then
+
+x = m:section(TypedSection, "client", translate("Limit Traffic Rate By Mac Address"))
```

Comment:
Can you please indent the code with a tab?
I know that the code is structured differently, but for clarity it would be good.
Maybe in a second commit you can adjust the indentation for the other if statements ass well?

Reply:
Thanks Florian! I will do another revision.

Reply:
Hi Florian,
I did some formatting for the new code -- I will create another pull request fixing other indentation issues once this pull request is merged.
Thanks again.
-- Tong